README.md

```diff
@@ -6,22 +6,91 @@
 [![codecov](https://codecov.io/gh/ZcashFoundation/zebra/branch/main/graph/badge.svg)](https://codecov.io/gh/ZcashFoundation/zebra)
 ![License](https://img.shields.io/badge/license-MIT%2FApache--2.0-blue.svg)
 
-Hello! I am Zebra, an ongoing Rust implementation of a Zcash node.
+[Zebra](https://zebra.zfnd.org/) is the Zcash Foundation's independent,
+consensus-compatible implementation of the Zcash protocol, currently under
+development.  Please [join us on Discord](https://discord.gg/na6QZNd) if you'd
+like to find out more or get involved!
 
-Zebra is a work in progress.  It is developed as a collection of `zebra-*`
-libraries implementing the different components of a Zcash node (networking,
-chain structures, consensus rules, etc), and a `zebrad` binary which uses them.
+Unlike `zcashd`, which originated as a Bitcoin Core fork and inherited its
+monolithic architecture, Zebra has a modular, library-first design, with the
+intent that each component can be independently reused outside of the `zebrad`
+fullnode.  For instance, the `zebra-network` crate containing the network stack
+can also be used to implement anonymous transaction relay, network crawlers, or
+other functionality, without requiring a full node.
 
-Most of our work so far has gone into `zebra-network`, building a new
-networking stack for Zcash, `zebra-chain`, building foundational data
-structures, `zebra-consensus`, implementing consensus rules, and
-`zebra-state`, providing chain state.
+Our first goal is to be able to participate in the network and replicate the
+Zcash chain state, and we intend to ship an alpha before the end of 2020 with
+this functionality.  In 2021, we intend to add RPC support and wallet
+integration.  This phased approach allows us to test the independent
```

Comment:
I think these are reasonable timeframes - but can we double-check the pre-alpha milestones before committing to end 2020?

For example, if there's a lot of detailed or unexpected work, or if one of the developers isn't working on Zebra for a while, we might actually finish early in Q1 2021.

It would be better to commit to a deadline that we're almost certain we could meet, and if everything goes well, we can announce an early release. (Rather than committing to an unrealistic deadline, then pushing it back.)

Reply:
I think we're certain to ship an alpha that can replicate chain state before the end of the year.  Per your suggestion, let's spend time this week on scoping to make sure we can do this.